typescript/infra/config/environments/mainnet3/rebalancer/USDC/lumia-config.yaml

```diff
@@ -0,0 +1,35 @@
+warpRouteId: USDC/lumia
+strategy:
+  rebalanceStrategy: minAmount
+  chains:
+    base:
+      minAmount:
+        min: 2000
+        target: 2500
+        type: 'absolute'
+      bridgeLockTime: 2400 # 40 mins in seconds
+      bridge: '0x5C4aFb7e23B1Dc1B409dc1702f89C64527b25975'
+
+    ethereum:
+      minAmount:
```

Comment:
I suspect the minimum should probably be a bit higher here, but happy to start with this